src/components/panel/panel.js

```diff
@@ -1051,6 +1094,7 @@ MdPanelRef.prototype._configurePosition = function() {
   this._panelEl.css('bottom', positionConfig.getBottom(this._panelEl));
   this._panelEl.css('left', positionConfig.getLeft(this._panelEl));
   this._panelEl.css('right', positionConfig.getRight(this._panelEl));
+  this._panelEl.css('transform', positionConfig.getTransform());
```

Comment:
One thing to keep in mind is that an element can only have one `transform` value, so setting it directly will override any transforms used for animations.